migrations/v4.js

```diff
@@ -0,0 +1,30 @@
+import { describe, whereContent, whereFromPlugin, mutateContent, checkContent, updatePlugin } from 'adapt-migrations';
+
+describe('adapt-contrib-blank - v2.0.0 > v4.1.3', async () => {
+  let blanks;
+
+  whereFromPlugin('adapt-contrib-blank - from v2.0.0', { name: 'adapt-contrib-blank', version: '<=4.1.3' });
+
+  whereContent('adapt-contrib-blank - where blank', async content => {
+    blanks = content.filter(({ _component }) => _component === 'blank');
+    if (blanks) return true;
```

Comment:
As per Ollie's suggestion (https://github.com/adaptlearning/adapt-contrib-accordion/pull/159#discussion_r1935417382) to condense these checks slightly suggested tweek
```suggestion
    return blanks.length
```